hcloud/schema_gen.go

```diff
@@ -0,0 +1,877 @@
+	SchemaFromLoadBalancer(*LoadBalancer) schema.LoadBalancer
+
+	// goverter:map Prices Pricings
+	LoadBalancerTypeFromSchema(schema.LoadBalancerType) *LoadBalancerType
+
+	// goverter:map Pricings Prices
+	SchemaFromLoadBalancerType(*LoadBalancerType) schema.LoadBalancerType
+
+	// goverter:map PriceHourly Hourly
+	// goverter:map PriceMonthly Monthly
+	LoadBalancerTypeLocationPricingFromSchema(schema.PricingLoadBalancerTypePrice) LoadBalancerTypeLocationPricing
+
+	// goverter:map Hourly PriceHourly
+	// goverter:map Monthly PriceMonthly
+	SchemaFromLoadBalancerTypeLocationPricing(LoadBalancerTypeLocationPricing) schema.PricingLoadBalancerTypePrice
+
+	LoadBalancerServiceFromSchema(schema.LoadBalancerService) LoadBalancerService
+
+	SchemaFromLoadBalancerService(LoadBalancerService) schema.LoadBalancerService
+
+	LoadBalancerServiceHealthCheckFromSchema(*schema.LoadBalancerServiceHealthCheck) LoadBalancerServiceHealthCheck
+
+	SchemaFromLoadBalancerServiceHealthCheck(LoadBalancerServiceHealthCheck) *schema.LoadBalancerServiceHealthCheck
+
+	LoadBalancerTargetFromSchema(schema.LoadBalancerTarget) LoadBalancerTarget
+
+	SchemaFromLoadBalancerTarget(LoadBalancerTarget) schema.LoadBalancerTarget
+
+	// goverter:map ID Server
+	LoadBalancerTargetServerFromSchema(schema.LoadBalancerTargetServer) LoadBalancerTargetServer
+
+	// goverter:map Server ID
+	SchemaFromLoadBalancerServerTarget(LoadBalancerTargetServer) schema.LoadBalancerTargetServer
+
+	LoadBalancerTargetHealthStatusFromSchema(schema.LoadBalancerTargetHealthStatus) LoadBalancerTargetHealthStatus
+
+	SchemaFromLoadBalancerTargetHealthStatus(LoadBalancerTargetHealthStatus) schema.LoadBalancerTargetHealthStatus
+
+	CertificateFromSchema(schema.Certificate) *Certificate
+
+	SchemaFromCertificate(*Certificate) schema.Certificate
+
+	PaginationFromSchema(schema.MetaPagination) Pagination
+
+	SchemaFromPagination(Pagination) schema.MetaPagination
+
+	// goverter:ignore response
+	// goverter:ignore DetailsRaw
+	// goverter:map Details | errorDetailsFromSchema
+	ErrorFromSchema(schema.Error) Error
+
+	// goverter:ignore response
+	// goverter:map Details | schemaFromErrorDetails
+	// goverter:map Details DetailsRaw | rawSchemaFromErrorDetails
+	SchemaFromError(Error) schema.Error
+
+	// goverter:map . Image | imagePricingFromSchema
+	// goverter:map . FloatingIP | floatingIPPricingFromSchema
+	// goverter:map . FloatingIPs | floatingIPTypePricingFromSchema
+	// goverter:map . PrimaryIPs | primaryIPPricingFromSchema
+	// goverter:map . Traffic | trafficPricingFromSchema
+	// goverter:map . ServerTypes | serverTypePricingFromSchema
+	// goverter:map . LoadBalancerTypes | loadBalancerTypePricingFromSchema
+	// goverter:map . Volume | volumePricingFromSchema
+	PricingFromSchema(schema.Pricing) Pricing
+
+	// goverter:map PriceHourly Hourly
+	// goverter:map PriceMonthly Monthly
+	serverTypePricingFromSchema(schema.PricingServerTypePrice) ServerTypeLocationPricing
```

Comment:
How did you decide which methods on the interface should be exported vs private?